### PR TITLE
Index curator coordinate access

### PR DESCRIPTION
**Pull request by kueda**
One way to address the geoprivacy issue @tiwane raised: https://inaturalist.slack.com/archives/C02F8RN5Q/p1499397827435545

### CODE REVIEW

File: app/es_indices/observation_index.rb [hunk 1]
```diff
@@ -168,7 +168,17 @@ def as_indexed_json(options={})
           project_observations.select{ |po| po.curator_identification_id.nil? }.
             map(&:project_id)).compact.uniq,
         project_observations: (indexed_project_ids || project_observations).map{ |po|
-          { project_id: po[:project_id], user_id: po[:user_id], uuid: po[:uuid] } },
```
Comment (pleary) [inline app/es_indices/observation_index.rb]: We shouldn't touch this right now. Node will fetch project observation preferences dynamically from the DB, so you don't need to add anything to the index.

I'm currently working on adding preferences to the index, and lots of other stuff, but that's not ready right now. So for this fix you should continue to use the old way, which means you don't need to change anything on the Rails end

File: app/es_indices/observation_index.rb [hunk 2]
```diff
@@ -240,12 +250,26 @@ def self.prepare_batch_for_index(observations, options = {})
     # fetch all project_ids store them in `indexed_project_ids`
     if options.blank? || options[:projects]
       connection.execute("
-        SELECT observation_id, project_id, curator_identification_id, uuid, user_id
```
Comment (pleary) [inline app/es_indices/observation_index.rb]: The changes here can be removed too. You won't need to change anything in `observation_index`

File: spec/lib/elastic_model/indices/observation_index_spec.rb [hunk 3]
```diff
@@ -162,6 +162,18 @@
     expect( json[:owners_identification_from_vision] ).to be true
   end
 
+  it "indexes project observation curator_coordinate_access in bulk" do
```
Comment (pleary) [inline spec/lib/elastic_model/indices/observation_index_spec.rb]: This can also be removed

**Comment by kueda** [comment]
Pref. loading happens from the DB in the API, Patrick is working on more sweeping index changes.

**Pull request closed without merging**